.config/requirements.in

```diff
@@ -8,7 +8,7 @@ jsonschema>=4.10.0 # MIT, version needed for improved errors
 packaging>=21.3 # Apache-2.0,BSD-2-Clause
 pyyaml>=5.4.1 # MIT (centos 9 has 5.3.1)
 rich>=12.0.0 # MIT
-ruamel.yaml>=0.17.24,<0.18 # MIT, next version is planned to have breaking changes
+ruamel.yaml>=0.17.0,<0.18 # MIT, next version is planned to have breaking changes
```

Comment:
That's scary. We touch some ruamel.yaml internals, and I don't know if they exist in older versions.